frontend/lexer.py

```diff
@@ -340,6 +394,10 @@ def LookPastSpace(self, lex_mode):
         # type: (lex_mode_t) -> Id_t
         return self.line_lexer.LookPastSpace(lex_mode)
 
+    def LookAheadDParens(self, shift_back):
```

Comment:
naming -- I think that `shift_back` is similar to `LookAheadFuncParens(unread)`

Although I think neither name is that good ... maybe it should be `num_bytes_back` or something ?   in both cases


```
        if self.cur_token.id == Id.Op_LParen:  # saw funcname(
            return self.lexer.LookAheadFuncParens(1)  # go back one char

        elif self.cur_token.id == Id.WS_Space:  # saw funcname WHITESPACE
            return self.lexer.LookAheadFuncParens(0)
```